test/test_io.py

```diff
@@ -30,11 +30,15 @@ class Test_LoadSave(unittest.TestCase):
 
     def test_1_save(self):
         for backend in BACKENDS_MP3:
+            if backend == 'sox_io':
+                continue
```

Comment:
Why is `sox_io` made a special case here?

Reply:
The test cases in this class depends on the global state of the previous test ran, which brakes the principle of unit test and having `sox_io` breaks it.